packages/core/integration-tests/test/sourcemaps.js

```diff
@@ -2,8 +2,65 @@ const assert = require('assert');
 const fs = require('@parcel/fs');
 const path = require('path');
 const mapValidator = require('sourcemap-validator');
+const SourceMap = require('parcel-bundler/src/SourceMap');
```

Comment:
Is importing an "internal" file from a different package acceptable?

Reply:
I'm using it to test the sourcemaps:
```js
let index = map.findClosestGenerated(
  generatedPosition.line,
  generatedPosition.column
);
```

Reply:
In that case it's probably fine